com.unity.render-pipelines.universal/Editor/UniversalRendererDataEditor.cs

```diff
@@ -10,16 +10,24 @@ public class UniversalRendererDataEditor : ScriptableRendererDataEditor
             public static readonly GUIContent PostProcessLabel = new GUIContent("Data", "The asset containing references to shaders and Textures that the Renderer uses for post-processing.");
-            public static readonly GUIContent FilteringLabel = new GUIContent("Filtering", "Controls filter rendering settings for this renderer.");
+            public static readonly GUIContent FilteringSectionLabel = new GUIContent("Filtering", "Settings that controls and define which layers the renderer draws.");
             public static readonly GUIContent OpaqueMask = new GUIContent("Opaque Layer Mask", "Controls which opaque layers this renderer draws.");
             public static readonly GUIContent TransparentMask = new GUIContent("Transparent Layer Mask", "Controls which transparent layers this renderer draws.");
-            public static readonly GUIContent RenderingLabel = new GUIContent("Rendering", "Settings related to rendering and lighting.");
+
+            public static readonly GUIContent RenderingSectionLabel = new GUIContent("Rendering", "Settings related to rendering and lighting.");
             public static readonly GUIContent RenderingModeLabel = new GUIContent("Rendering Path", "Select a rendering path.");
             public static readonly GUIContent DepthPrimingModeLabel = new GUIContent("Depth Priming Mode", "With depth priming enabled, Unity uses the depth buffer generated in the depth prepass to determine if a fragment should be rendered or skipped during the Base Camera opaque pass. Disabled: Unity does not perform depth priming. Auto: If there is a Render Pass that requires a depth prepass, Unity performs the depth prepass and depth priming. Forced: Unity performs the depth prepass and depth priming.");
             public static readonly GUIContent DepthPrimingModeInfo = new GUIContent("On Android, iOS, and Apple TV, Unity performs depth priming only in the Forced mode. On tiled GPUs, which are common to those platforms, depth priming might reduce performance when combined with MSAA.");
             public static readonly GUIContent RenderPassLabel = new GUIContent("Native RenderPass", "Enables URP to use RenderPass API. Has no effect on OpenGLES2");
+
+            public static readonly GUIContent RenderPassSectionLabel = new GUIContent("RenderPass", "This section contains properties related to render passes.");
+            public static readonly GUIContent ShadowsSectionLabel = new GUIContent("Shadows", "This section contains properties related to rendering shadows.");
+            public static readonly GUIContent PostProcessingSectionLabel = new GUIContent("Post-processing", "This section contains properties related to rendering post-processing.");
+
+            public static readonly GUIContent OverridesSectionLabel = new GUIContent("Overrides", "This section contains Render Pipeline properties that this Renderer overrides.");
```

Comment:
Change them to `EditorGUIUtility.TrTextContent` instead of `new GUIContent` as we need the localization.

Reply:
fixed.